src/subdomains/generic/user/models/user/dto/user-dto.mapper.ts

```diff
@@ -23,7 +23,7 @@ export class UserDtoMapper {
       },
       volumes: this.mapVolumes(userData),
       addresses: userData.users
-        .filter((u) => !u.isBlockedOrDeactivated && !u.wallet.isKycClient)
+        .filter((u) => !userData.isBlockedOrDeactivated && !u.isBlockedOrDeactivated && !u.wallet.isKycClient)
         .map((u) => this.mapAddress(u)),
       activeAddress: activeUser && this.mapAddress(activeUser),
```

Comment:
I'm not sure if it makes sense to filter by user-data status here 🤔  these users all have the same user-data. And the user can't login anymore.